eth-node/types/bytes.go

```diff
@@ -39,3 +39,12 @@ func Hex2Bytes(str string) []byte {
 	h, _ := hex.DecodeString(str)
 	return h
 }
+
+// ToHex returns the hex string representation of bytes with 0x prefix.
+func ToHex(bytes []byte) string {
+	str := Bytes2Hex(bytes)
+	if !has0xPrefix(str) {
+		str += "0x" + str
+	}
+	return str
```

Comment:
```suggestion
	return "0x" + Bytes2Hex(bytes)
```

`Bytes2Hex` never returns with prefix, does it?